src/main/java/repository/strings/Storage.java

```diff
@@ -0,0 +1,91 @@
+
+    public static final String YOUR_CHOICE_STRING = "Вы выбрали: ";
+    public static final String EAES_STRING = "автомобили из ЕАЭС";
+    public static final String OTHER_COUNTRIES_STRING = "автомобили из иных стран";
+    public static final String PHYSICAL_STRING = ", физическое лицо";
+    public static final String JURIDICAL_STRING = ", юридическое лицо";
+    public static final String LESS_3_YEARS_STRING = ", автомобиль младше 3 лет";
+    public static final String BETWEEN_3_AND_7_YEARS_STRING = ", автомобиль от 3 до 7 лет";
+    public static final String MORE_7_YEARS_STRING = ", автомобиль старше 7 лет";
+    public static final String GAS_STRING = ", бензиновый или гибридный двигатель";
+    public static final String ELECTRIC_STRING = ", электродвигатель";
+    public static final String PRICE_STRING = "Цена: ";
+    public static final String VOLUME_LESS_1000_STRING = " объем не более 1000 куб. см";
+    public static final String VOLUME_BETWEEN_1000_2000_STRING = " объем от 1000 куб. см до 2000 куб. см";
+    public static final String VOLUME_BETWEEN_2000_3000_STRING = " объем от 2000 куб. см до 3000 куб. см";
+    public static final String VOLUME_BETWEEN_3000_3500_STRING = " объем от 3000 куб. см до 3500 куб. см";
+    public static final String VOLUME_MORE_3500_STRING = " объем свыше 3500 куб. см";
+    public static final String BYN_STRING = " BYN\n";
+    public static final String FAREWELL_STRING = "\nДополнительная информация.\n custom.by - оформление утильсбора;  \n /start - перезапустить калькулятор.";
+
+    public static final String START = "/start";
+    public static final String EAES = "/eaes";
+    public static final String OTHER_COUNTRIES = "/other";
+    public static final String PHYSICAL = "/physical";
+    public static final String JURIDICAL = "/juridical";
+    public static final String LESS_3_YEARS = "/less_3";
+    public static final String BETWEEN_3_AND_7_YEARS = "/3_7_years";
+    public static final String MORE_7_YEARS = "/more_7";
+    public static final String GASOLINE = "/gasoline";
+    public static final String ELECTRIC = "/electric";
+    public static final String VOLUME_LESS_1000 = "/less_1000";
+    public static final String VOLUME_BETWEEN_1000_2000 = "/1000_2000";
+    public static final String VOLUME_BETWEEN_2000_3000 = "/2000_3000";
+    public static final String VOLUME_BETWEEN_3000_3500 = "/3000_3500";
+    public static final String VOLUME_MORE_3500 = "/more_3500";
+
+    //passenger cars - EAES or other countries, physical or juridical with electric engines
+    public static final String PRICE_PASSENGER_LESS_3_YEARS = "544.5";
+    public static final String PRICE_PASSENGER_3_TO_7_YEARS = "816.7";
+    public static final String PRICE_PASSENGER_OLDER_7_YEARS = "1225.1";
+
+    //other countries - gasoline engine prices
+    public static final String PRICE_PASSENGER_OTHER_GASOLINE_1000_LESS_3_YEARS = "1652.2";
```

Comment:
I recommend to move this constant to independent class

Reply:
Done.